pkg/maps/bwmap/bwmap.go

```diff
@@ -66,3 +66,14 @@ func Delete(Id uint16) error {
 	return ThrottleMap.Delete(
 		&EdtId{Id: uint64(Id)})
 }
+
+func SilentDelete(Id uint16) error {
+	_, err := ThrottleMap.SilentDelete(
+		&EdtId{Id: uint64(Id)})
+
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
nit: just `return err` here?

Reply:
Obviously! Fixed and re-pushed.
https://github.com/cilium/cilium/compare/909261162597ce88bb53e44f253c6f8fd4418097..280de074362d21880b247c95674c33215175671a